### PR TITLE
Fix doc and clippy warning.

### PR DESCRIPTION
**Pull request by AurelienFT**

### CODE REVIEW

**Comment by AurelienFT** [comment]
Fix #2771

File: massa-graph/src/block_graph.rs [hunk 1]
```diff
@@ -591,7 +591,7 @@ impl BlockGraph {
     pub fn export_bootstrap_graph(&self) -> Result<BootstrapableGraph> {
         let mut required_final_blocks: Set<_> = self.list_required_active_blocks()?;
         required_final_blocks.retain(|b_id| {
-            if let Some(BlockStatus::Active(a_block)) = self.block_statuses.get(&b_id) {
+            if let Some(BlockStatus::Active(a_block)) = self.block_statuses.get(b_id) {
```
Comment (Eitu33) [inline massa-graph/src/block_graph.rs:594]: fixed this here already https://github.com/massalabs/massa/pull/2765

Comment (damip) [reply massa-graph/src/block_graph.rs:594]: is there a conflict ? I think it's ok, we can just merge this one

**Review by damip — APPROVED** [review]

File: massa-graph/src/block_graph.rs (hunk 1 above)
Comment (AurelienFT) [reply massa-graph/src/block_graph.rs:594]: Yes it's ok

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**